src/style.css

```diff
@@ -0,0 +1,147 @@
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+
+  --color-main-text: #060b35;
+  --color-add-text: #616070;
+  --color-accent: #00acdc;
+  --color-light: #f3f3f3;
+  --main-radius: 5px;
+}
+
+.page__content {
+  margin: 0;
+}
+
+.title {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid var(--color-light);
+  border-radius: var(--main-radius);
+  padding: 32px 16px 15px;
+}
+
+.card__img {
+  width: 159px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__img img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-main-text);
+  margin-bottom: 4px;
+}
+
+.card__code {
+  color: var(--color-add-text);
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.card__review {
+  margin-bottom: 24px;
+}
+
+.review {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}

```

Comment:
Styles for review BEM block should be in review.css file, and check if you really need this BEM block, or you can make it just the element of the card block